docs/framework/data/adonet/sql/linq/linq-to-sql-queries.md

```diff
@@ -6,7 +6,7 @@ ms.assetid: f4897aaa-7f44-4c20-a471-b948c2971aae
 ---
 # LINQ to SQL Queries
 
-You define [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] queries by using the same syntax as you would in LINQ. The only difference is that the objects referenced in your queries are mapped to elements in a database. For more information, see [Introduction to LINQ Queries (C#)](../../../../../csharp/programming-guide/concepts/linq/introduction-to-linq-queries.md).  
+You define [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] queries by using the same syntax as you would in LINQ. The only difference is that the objects referenced in your queries are mapped to elements in a database. For more information, see [Introduction to LINQ Queries (C#)](../../../../../csharp/linq/get-started/introduction-to-linq-queries.md).  
```

Comment:
Not worth changing now, but when relative links get this deep it might make sense to go with absolute links -- `/includes/vbtecdlinq-md.md` instead of `../../../../../../includes/vbtecdlinq-md.md`, `/csharp/linq/get-started/introduction-to-linq-queries.md` instead of `../../../../../csharp/linq/get-started/introduction-to-linq-queries.md`, etc.